app/Http/Controllers/CompetitionsController.php

```diff
@@ -206,4 +209,42 @@ public function leaderboard(Competition $competition)
 
         return view('competitions.leaderboard', compact('competition', 'leaderboard', 'pending', 'flagged'));
     }
+
+    /**
+     * Get the featured reportback form.
+     *
+     * @param  \Gladiator\Models\Competition  $competition
+     * @param  \Gladiator\Models\Message  $message
+     * @return \Illuminate\Http\Response
+     */
+    public function editFeatureReportback(Competition $competition, Message $message)
+    {
+        $reportback = FeaturedReportback::where('competition_id', '=', $competition->id)->where('message_id', '=', $message->id)->first();
+
+        return view('competitions.featured_reportback', compact('competition', 'message', 'reportback'));
+    }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param  \Gladiator\Requests\FeaturedReportbackRequest  $request
+     * @param  \Gladiator\Models\Competition  $competition
+     * @param  \Gladiator\Models\Message  $message
+     * @return \Illuminate\Http\Response
+     */
+    public function updateFeaturedReportback(FeaturedReportbackRequest $request, Competition $competition, Message $message)
+    {
+        $reportback = FeaturedReportback::where('competition_id', '=', $competition->id)->where('message_id', '=', $message->id)->first();
+
+        if (! isset($reportback)) {
+            $reportback = new FeaturedReportback;
+            $reportback->competition_id = $competition->id;
+            $reportback->message_id = $message->id;
+            $reportback->save();
```

Comment:
This could all be pulled out into a `createFeaturedReportback()` method that could be called from here, so we'd have access to it outside the update call. Not crucial for merge, just noting for future. Kinda also feel like a bunch of this could be pulled out into its own `FeaturedReportbacksController` class, but it's debatable since it's part of the `Competitions`... topic for a later nerd discussion :neckbeard: